CONTRIBUTING.md

```diff
@@ -0,0 +1,68 @@
+# Contributing to the Self Hosted Show Wiki
+
+
+## Objectives of This Wiki
+
+In order to contribute well, you need to understand what the objectives are behind this wiki.
+
+What we are trying to create:
+- A compilation of tools, tips and tricks for self-hosting specific applications.
+- A place where beginners and experts can learn about how to quickly, efficiently and securely deploy applications on their own server(s).
+- Snippets that people can quickly reference in deploying applications.
+- A hub for quality documentation (to the extent possible).
+
+What we are not trying to do:
+- Be another large list of self-hosted tools. See [awesome-selfhosted](https://github.com/awesome-selfhosted/awesome-selfhosted) for a comprehensive list of self-hosted applications.
+- Be a hub for script kitties. People should _learn_ from documentation here not just come here to copy/paste. Snippets are important but should be documented properly.
+- Insecure, sloppy or any undesirable systems administration practices such as:
+  - Executing remote code by `curl`-ing an untrusted script and piping it to `sudo bash`: `curl https://example.com/awesomescript.sh | sudo bash`
+  - Installing additional bloat within a Docker container
+  - Etc.
+
+
+We want to encourage the use of dev/ops tools such as Docker, Ansible, SALT Stack, etc. for increased ease of use and for automating/standardizing as much as possible.
+
+
+## Directory Layout:
+
+The `docs` folder in the root of the repository is where wiki content should be placed.
+
+The following example directory layout should be adhered to when contributing:
+
+`docs/category/application/application.md`
```

Comment:
For the main landing page of a given application, the pattern should probably be:

```suggestion
`docs/category/application/index.md`
```

Reply:
That works for me. I didn't change it initially since the index.md suggestion was in relation to fixing the duplicate items in the menu. But thinking about this generally, it _does_ make sense to have the main landing page for a topic to be `index.md`.

Reply:
:+1: